src/evaluation/move_vote_map.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "movegen/move_types.h"
+
+namespace evaluation {
+
+template<size_t maxSize>
+class MoveVoteMap {
+
+    using MoveVotePair = std::pair<movegen::Move, uint8_t>;
+
+public:
+    MoveVoteMap() = default;
+
+    MoveVoteMap(std::initializer_list<MoveVotePair> init)
+    {
+        assert(init.size() < maxSize);
+
+        for (auto& entry : init) {
+            m_entries.at(m_size++) = std::move(entry);
+        }
+    }
+
+    constexpr void insertOrIncrement(movegen::Move ext_move, uint8_t ext_vote)
```

Comment:
We don't use snake case :wink: 
```suggestion
    constexpr void insertOrIncrement(movegen::Move extMove, uint8_t extVote)
```